assets/xml/objects/object_hakach_objects.xml

```diff
@@ -0,0 +1,28 @@
+<Root>
+    <File Name="object_hakach_objects" Segment="6">
+        <DList Name="gBotwHoleTrap1DL" Offset="0x01B0"/>
```

Comment:
I'd rather this either just be Well or BottomOfTheWell since botw isn't immediately obvious what it's for, and well, also sounds like another game 😛

Reply:
the botw acronym was in use way before the existence of breath of the wild, to be fair.
and id say its a pretty common acronym for this game, dont see an issue personally

Reply:
I think Well still gets the point across though without having to think about what the acronym means.

Reply:
We have been using either the whole name or acronyms for dungeons when naming assets, so Botw is fine imo.